contributions/covid19/tommysam-jii/COVID19_COURSE_INFO.md

```diff
@@ -0,0 +1,104 @@
+
+<a name="func"> </a>
+
+### Functionality
+
+#### COVID-19 Statistics
+If you visit the [URL](https://dd2482-covid19-course-info.firebaseapp.com) for the application, you will enter the ``Home`` screen for the app. At the home screen, you will see the current numbers and statistics for the virus COVID-19, both in Sweden and in the rest of the world. The application fetch the COVID-19 data from the [NovelCOVID](https://github.com/novelcovid/api) API, so the numbers might be a little bit different compared to numbers on the [Public Health Agency of Sweden](https://www.folkhalsomyndigheten.se/smittskydd-beredskap/utbrott/aktuella-utbrott/covid-19/bekraftade-fall-i-sverige/) web page.
+
+If you want to see the current numbers for another country, you can add the country to the start page by clicking on the ``More`` symbol at the bottom of the screen and then ``Add countries``. In ``Add countries``, you can search for another country and then add their current COVID-19 data to the home screen of the application. 
+
+#### KTH Courses
+If you click on the button ``Courses`` at the bottom of the screen, you will get a list of some of the courses at KTH. Each course object contains the course code, the name of the course and how the course will be examined now due to the COVID-19. If you click on one of the courses, you will also get some more information about the picked course. If you want to update the information for the picked course, you can click on the ``three dots`` in the right corner at the top of the screen. 
+
+If you want to submit a new course, click on the ``Submit`` button at the bottom of the screen. We have added some restrictions on how the new course submission should be structured:
+* Course code should be 6 characters long and start with two letters
+* You need to add some information about how the course is examined
+* The course information cannot be longer than 60 characters.
+
+If you don't fulfill these criteria, you will receive an error message with the information you need to add or change to be able to successfully make a submission.
+
+<a name="dev"> </a>
+
+### Devops Tools
+
+#### Jenkins 
+One of the intended learning outcomes for the course Automated Software Testing and DevOps DD2482, is that the student should be able to setup and deploy continuous integration. Therefore, we decided to try to create our own Jenkins server and integrate it with our COVID-19 project. 
+
+We are running Jenkins as Docker container from the Blue Ocean Docker image. We have created two different pipelines. One is the main pipeline that are connected to the original GitHub repository for our project. The other one is a pipeline connected to the a forked copy of the original repository, which we use to test changes in the set up of the pipeline, or when we try to build new stages for the original pipeline.
```

Comment:
Add link to your Dockerfile. Explain it in detail.

Reply:
I am a totally beginner to Docker, so correct me if I am wrong. We don't have a Dockerfile for this project, since we are only using the existing image jenkinsci/blueocean for our Jenkins machine and pipeline. We have used Jenkins own tutorial for the set up for our app: https://jenkins.io/doc/tutorials/build-a-node-js-and-react-app-with-npm/#run-jenkins-in-docker 
If you want, we could add a link to this documentation to describe how we managed to set up our small pipeline. Would that work instead?